Makefile

```diff
@@ -121,42 +113,62 @@ tidy:
 
 .PHONY: build
 build: $(UNZIP_DEST)
-	$(CPREFIX) GOOS=$(GOOS) GOARCH=$(GOARCH) CGO_ENABLED=1 go build -ldflags '$(LDFLAGS)'
+	CGO_CFLAGS="-I$(REAL_DEST)" \
+	CGO_LDFLAGS="-L$(REAL_DEST)" \
+	GOOS=$(GOOS) \
+	GOARCH=$(GOARCH) \
+	CGO_ENABLED=1 \
+	go build -ldflags '$(LDFLAGS)'
 
 .PHONY: test
 test: $(UNZIP_DEST)
-	LD_LIBRARY_PATH=$(REAL_DEST) go test -v -ldflags "-extldflags \"-L$(REAL_DEST) -led25519_bip32\"" ./...
+	CGO_CFLAGS="-I$(REAL_DEST)" \
+	CGO_LDFLAGS="-L$(REAL_DEST)" \
+	LD_LIBRARY_PATH=$(REAL_DEST) \
+	go test -v -count 1 -ldflags "-extldflags \"$(STATICLDFLAGS)\"" ./...
 
 .PHONY: test-tidy
-test-tidy:
+test-tidy: install
```

Comment:
`install` was not a dependent target of most of the others because its not a `no-op` if it was already executed.

This will install the tools used multiple times in a CI run. using them locally also becomes more tedious.

Reply:
fair point, will fix!